CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## Unreleased 
```

Comment:
I feel like we can remove `Unreleased` if there isn't anything in it.

Reply:
Fresh 👍